core/cmd/solana_transaction_commands_test.go

```diff
@@ -29,7 +29,7 @@ func TestShell_SolanaSendSol(t *testing.T) {
 	url := solanaClient.SetupLocalSolNode(t)
 	node := solcfg.Node{
 		Name: ptr(t.Name()),
-		URL:  utils.MustParseURL(url),
+		URL:  config.MustParseURL(url),
```

Comment:
Huh, weird that we even have this in multiple locations. Seems like Solana should be importing this from `chainlink-commons` right?